CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [1.0.17] - 2024-02-27
+
+### Fixed
```

Comment:
The `1.0.17` header will be added automatically during the next release, just keep the changes under the `unreleased` header. More info in [keepachangelog](https://keepachangelog.com/en/1.1.0/#effort) website
```suggestion
### Fixed
```